app/channels/room_channel.rb

```diff
@@ -13,6 +19,12 @@ def speak(data)
       user: current_user,
       text: message
     )
+
+    other_user = conversation.other_user(current_user)
+    return if conversation.connected_user(other_user)
+
+    conversation.increase_unread(other_user)
+    NotifyMessageJob.perform_later(other_user)
```

Comment:
I think the channel should not be responsible for this, maybe you can move this logic to the `Message` model or some service instead. WDYT @JuanfraM @GuilleLeopold @mdornel ?